airflow-core/src/airflow/jobs/triggerer_job_runner.py

```diff
@@ -600,11 +601,15 @@ def update_triggers(self, requested_trigger_ids: set[int]):
 
             new_trigger_orm = new_triggers[new_id]
 
-            # If the trigger is not associated to a task or an asset, this means the TaskInstance
+            # If the trigger is not associated to a task, an asset or a deadline, this means the TaskInstance
```

Comment:
```suggestion
            # If the trigger is not associated to a task, an asset, or a deadline, this means the TaskInstance
```

Sorry, can't help myself...